views/public/javascripts/mall-map.js

```diff
@@ -268,54 +260,129 @@ function mallMapJs(){
     // Handle the info panel back button.
     $('a.back-button').click(function (e) {
         e.preventDefault();
-        $('#info-panel').fadeToggle(200, 'linear');
-        $('#toggle-map-button + .back-button').hide();
+        $('#info-panel-container').fadeToggle(200, 'linear');
+        $('#toggle-map-button + .back-button + .next-button').hide();
```

Comment:
Can this be removed now that the next has moved below?

Reply:
fixed, just push the code